easybuild/easyconfigs/p/Python/Python-2.7.10-gimkl-2.11.5.eb

```diff
@@ -0,0 +1,121 @@
+name = 'Python'
+version = '2.7.10'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'gimkl', 'version': '2.11.5'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.2'
+scipyversion = '0.15.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('SQLite', '3.8.10.2'),
+    ('Tk', '8.6.4', '-no-X11'),
```

Comment:
hmm, apparently this hasn't been done consistently in other Python 2.7.10 easyconfigs... so let's sync this up in a follow up PR